src/ioq_sup.erl

```diff
@@ -54,9 +58,15 @@ get_ioq2_servers() ->
         list_to_atom("ioq_server_" ++ integer_to_list(I))
     end, lists:seq(1, erlang:system_info(schedulers))).
 
+get_all_ioq2_servers() ->
```

Comment:
Another option is to use following API

```erlang
ioq2_servers() -> 
    ioq2_servers_by_type(search) ++ ioq_sup:ioq2_servers_by_type(normal).

ioq2_servers_by_type(search) ->
    ...
ioq2_servers_by_type(normal) ->
    ...
```

However I wouldn't insist on changing it.

Reply:
Yeah I think something along these lines would be nice, but I think `ioq_sup:ioq2_servers_by_type(normal).` is a bit verbose to outside callers, so I'm going to leave it be for the moment. Let's revisit if we add anymore IOQ2 process types.